src/brisk.jl

```diff
@@ -0,0 +1,106 @@
+"""
+```
+brisk_params = BRISK([pattern_scale = 1.0])
+```
+
+| Argument | Type | Description |
+|----------|------|-------------|
+| `pattern_scale` | `Float64` | Scaling factor for the sampling window | 
+"""
+type BRISK <: Params
+    threshold::Float64
+    octaves::Int
+    pattern_scale::Float64
+    pattern_table::Vector{Vector{SamplePair}}
+    smoothing_table::Vector{Vector{Float16}}
+    orientation_weights::Vector{OrientationWeights}
+    short_pairs::Vector{OrientationPair}
+    long_pairs::Vector{OrientationPair}
```

Comment:
Since these variables are internal, I've replaced the parametric `BRISK{}` as this looks better. All of them are leaftypes so I guess it shouldnt matter. At some point, we should have redefine the `print` function for these types so that the internal variables do not get printed while displaying the result of calling the constructor.